src/battle_util.c

```diff
@@ -1567,6 +1568,65 @@ static bool32 EndTurnTerrain(u32 terrainFlag, u32 stringTableId)
     return FALSE;
 }
 
+#define BATTLE_WEATHER_MESSAGE_STOPPED  0
+#define BATTLE_WEATHER_MESSAGE_TURN     1
+#define BATTLE_WEATHER_ANIMATION        2
+static const u32 sBattleWeatherAttributes[][3] =
```

Comment:
Probably better as a struct

Reply:
Could you give me a hint on how I should structure it?

Reply:
Changed it to a struct. Let me know if this is what you imagined

(still no clean up)